mk/spksrc.cross-cmake.mk

```diff
@@ -114,14 +114,6 @@ $(CMAKE_TOOLCHAIN_PKG):
 cmake_pkg_toolchain:
 	@cat $(CMAKE_TOOLCHAIN_WRK) ; \
 	echo
-ifeq ($(strip $(CMAKE_USE_NASM)),1)
-ifeq ($(findstring $(ARCH),$(i686_ARCHS) $(x64_ARCHS)),$(ARCH))
-	@echo "# set assembly compiler" ; \
-	echo "set(ENABLE_ASSEMBLY $(ENABLE_ASSEMBLY))" ; \
-	echo "set(CMAKE_ASM_COMPILER $(CMAKE_ASM_COMPILER))" ; \
-	echo
-endif
-endif
```

Comment:
This is rather interesting... indeed cmake should auto-detect `nasm` as being in default PATH.